### PR TITLE
{SM-847] Updated Delete service account modal to be default size, not small

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## Type of change

```
- [ x ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/dialog/service-account-delete-dialog.component.html 
 - made the delete service account modal bigger by setting dialogSize to default
## Code changes

Updated modal size to Default instead of small

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/dialog/service-account-dialog.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <form [formGroup]="formGroup" [bitSubmit]="submit">
-  <bit-dialog dialogSize="small">
+  <bit-dialog dialogSize="default">
```
Comment (Thomas-Avery) [inline bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/dialog/service-account-dialog.component.html:2]: If we are updating the size of the creation/edit dialog for service accounts, we will probably want to update the project create/edit dialog too.

**Review by Thomas-Avery — CHANGES_REQUESTED** [review]

File: bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/dialog/service-account-dialog.component.html (hunk 1 above)
Comment (cd-bitwarden) [reply bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/dialog/service-account-dialog.component.html:2]: okay i updated project-dialog.component.html as well 😄

**Review by Thomas-Avery — APPROVED** [review]
LGTM

**Comment by cd-bitwarden** [comment]
Ready to be tested

**Pull request merged**